bin/fccanalysis

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+
+
+from config.FCCAnalysisRun import *
+
+if __name__ == "__main__":
+  import argparse
+  parser = argparse.ArgumentParser()
+  subparsers = parser.add_subparsers() 
+  parser_run = subparsers.add_parser('run', help="run a RDataFrame based FCC analysis")
+  run(parser, parser_run)
```

Comment:
```suggestion
if __name__ == "__main__":
  import argparse
  import sys
  parser = argparse.ArgumentParser()
  subparsers = parser.add_subparsers() 
  parser_run = subparsers.add_parser('run', help="run a RDataFrame based FCC analysis")
  if len(sys.argv)<2:
     print("for usage run fccanalyses --help")
     sys.exit(3)
  from config.FCCAnalysisRun import * #such that the dictionary is loaded only if the configuration is ok
  run(parser, parser_run)
```

Reply:
suggestion in case we run ```fccanalysis``` without any arguments, such that we trigger some help